src/hooks/useConsoleErrors.ts

```diff
@@ -2,23 +2,18 @@ import { useEffect } from 'react';
 import { IQuizProps } from '../types';
 
 const useConsoleErrors = (quizOptions: IQuizProps) => {
-  const { quizId, resultsPageOptions } = quizOptions;
+  const { quizId, resultsPageOptions, callbacks } = quizOptions;
   useEffect(() => {
     if (!quizId) {
       // eslint-disable-next-line no-console
       console.error('quizId is a required field of type string');
     }
 
-    if (!resultsPageOptions || Object.keys(resultsPageOptions).length === 0) {
```

Comment:
Removed resultsPageOptions since they're only required for the component interface and not for hooks.
Added to the docs though